drivers/bridge/bridge.go

```diff
@@ -446,14 +446,14 @@ func (d *driver) getNetwork(id types.UUID) (*bridgeNetwork, error) {
 	defer d.Unlock()
 
 	if id == "" {
-		return nil, types.BadRequestErrorf("invalid network id: %s", id)
+		return nil, InvalidNetworkIDError(id)
 	}
 
 	if nw, ok := d.networks[id]; ok {
 		return nw, nil
 	}
 
-	return nil, nil
+	return nil, NetworkNotFoundError(id)
```

Comment:
Thanks for this fix.
Please replace nil with types.NotFoundErrorf(....) instead.